docs/rofl/app.md

```diff
@@ -0,0 +1,484 @@
+the value of the `ORACLE_CONTRACT_ADDRESS` constant defined at the top of
+`main.rs`:
+
+```rust
+/// Address where the oracle contract is deployed.
+const ORACLE_CONTRACT_ADDRESS: &str = "0x1234845aaB7b6CD88c7fAd9E9E1cf07638805b20";
+```
+
+Make sure to use the contract address as output by the deployment script in the
+previous step.
+
+## Building the ROFL App
+
+In order to quickly build the ROFL app you can use the helpers provided by the
+Oasis CLI as follows:
+
+```bash
+oasis rofl build sgx --network testnet --paratime sapphire
+```
+
+This will build the required app binaries using `cargo` and bundle them into
+the Oasis Runtime Container (ORC) format suitable for deployment to Oasis nodes.
+By default, the resulting file will be called `rofl-oracle.orc`.
+
+:::info Reproducibility
+
+For audit reasons it is very important that ROFL app binaries can be reproduced
+from the given source code. This makes it possible to check that the right code
+is actually deployed. In order to support reproducible builds, please see the
+[Reproducibility chapter].
+
+[Reproducibility chapter]: https://github.com/oasisprotocol/oasis-sdk/blob/main/docs/runtime/reproducibility.md
+
+:::
+
+## Updating the ROFL App Policy
+
+Now that the app binaries are available, we need to update the policy with the
+correct cryptographic identity of the app. To obtain the identity of the app
+that was just built run:
+
+```bash
+oasis rofl identity rofl-oracle.orc
+```
+
+This should output something like the following:
+
+```
+yg1zKYeLa4QjEj08dkMtNn4JnUmNrpuov4QwC3X9ZBUJFtTP3jZWjgVaV8WjTmoAr2gcg9ymZYoUFM2y9rp1Jw==
+```
+
+This represents the cryptographic identity of the ROFL app. We now need to
+update the policy to ensure that only exact instances of the built app can
+successfully authenticate under our app ID. To do so, update the previously
+generated `policy.yml` as follows (using your own app identity):
+
+```yaml
+# Acceptable remote attestation quotes.
+quotes:
+    # Intel SGX/TDX PCS (DCAP) quotes.
+    pcs:
+        # Maximum age (in days) of the acceptable TCB infos.
+        tcb_validity_period: 30
+        # Minimum acceptable TCB evaluation data number. This ensures that TCB information
+        # provided by the TEE vendor is recent enough and includes relevant TCB recoveries.
+        min_tcb_evaluation_data_number: 17
+# Acceptable enclave cryptographic identities.
+enclaves:
+    - "yg1zKYeLa4QjEj08dkMtNn4JnUmNrpuov4QwC3X9ZBUJFtTP3jZWjgVaV8WjTmoAr2gcg9ymZYoUFM2y9rp1Jw=="
+# Acceptable nodes that can endorse the enclaves.
+endorsements:
+    - any: {} # Any node can endorse.
+# Who is paying the transaction fees on behalf of the enclaves.
+fees: endorsing_node # The endorsing node is paying via a fee proxy.
+# How often (in epochs) do the registrations need to be refreshed.
+max_expiration: 3
+```
+
+Then to update the on-chain policy, run (using _your own app identifier_ instead
+of the placeholder `rofl1qp55evqls4qg6cjw5fnlv4al9ptc0fsakvxvd9uw`):
```

Comment:
It's usually better to use an invalid placeholder, so that validation or command fails if the user just copies it.